src/Propel/Generator/Manager/MigrationManager.php

```diff
@@ -264,13 +265,21 @@ public function updateLatestMigrationTimestamp(string $datasource, int $timestam
     {
         $platform = $this->getPlatform($datasource);
         $conn = $this->getAdapterConnection($datasource);
+
+        $this->modifyMigrationTableIfOutdated($conn, $platform);
+
         $sql = sprintf(
-            'INSERT INTO %s (%s) VALUES (?)',
+            'INSERT INTO %s (%s, %s) VALUES (?, ?)',
             $this->getMigrationTable(),
-            $platform->doQuoting('version'),
+            $platform->doQuoting(static::COL_VERSION),
+            $platform->doQuoting(static::COL_EXECUTION_DATETIME),
         );
+
+        $executionDatetime = date('Y-m-d H:i:s');
```

Comment:
Could you please move the date template to the protected const?

Reply:
@yaroslav-spryker, moved.